### PR TITLE
[Term Entry] Lua Tables getn()

### PR DESCRIPTION
**Pull request by badday182**
### Description

- Added Table.Getn() term for Lua
- Closes issue #3168 #
### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/lua/concepts/tables/terms/getn/getn.md
Comment (caupolicandiaz) [inline content/lua/concepts/tables/terms/getn/getn.md:13]: Please update the spacing in the entry. There should never be multiple empty lines between content, and there should always be one empty line after a header or code block

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@badday182 I've added a couple of initial comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by badday182** [comment]
@caupolicandiaz  Thank you for your feedback, I finished making changes:
- updated the path and the entry to Luau
- updated the spacing in the entry

File: content/luau/concepts/tables/terms/getn/getn.md
Comment (caupolicandiaz) [inline content/luau/concepts/tables/terms/getn/getn.md:15]:
```suggestion
> **Note:** This function has been deprecated. It is recommended that new projects use `#t` instead.
```

Comment (caupolicandiaz) [reply content/luau/concepts/tables/terms/getn/getn.md:15]: move this after the description/summary

File: content/luau/concepts/tables/terms/getn/getn.md
Comment (caupolicandiaz) [inline content/luau/concepts/tables/terms/getn/getn.md:28]:
```suggestion
```lua
```

File: content/luau/concepts/tables/terms/getn/getn.md
Comment (caupolicandiaz) [inline content/luau/concepts/tables/terms/getn/getn.md:36]:
```suggestion
```shell
```

File: content/luau/concepts/tables/terms/getn/getn.md
Comment (caupolicandiaz) [inline content/luau/concepts/tables/terms/getn/getn.md:60]:
```suggestion
```shell
```

File: content/luau/concepts/tables/terms/getn/getn.md
Comment (caupolicandiaz) [inline content/luau/concepts/tables/terms/getn/getn.md:80]:
```suggestion
```shell
```

File: content/luau/concepts/tables/terms/getn/getn.md
Comment (caupolicandiaz) [inline content/luau/concepts/tables/terms/getn/getn.md:45]: this output should also be in a separate ```shell block

File: content/luau/concepts/tables/terms/getn/getn.md
Comment (caupolicandiaz) [inline content/luau/concepts/tables/terms/getn/getn.md:68]: There should be an empty line after the example header

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@badday182 I've added some additional comments.

**Comment by badday182** [comment]
@caupolicandiaz i've finished making changes:
- moved "Note:" to the end of description/summary
- chenged "luau" to "lua"
- chenged/added ```shell block

**Comment by badday182** [comment]
@caupolicandiaz i've made several corrections:
- changed and moved note about deprecated function to the end of description/summary
- luau block changed to lua block
- used shell block where it's correct

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@badday182 There are a number of additional edits that got mixed in with this PR. There should only be one file here, please revert the other changes. You should be able to use the following syntax-
git checkout main -- filepath

**Review by caupolicandiaz — APPROVED** [review]
@badday182 This is ready for a second review.  However, there continue to be a pair of files that should not be part of this PR

**Comment by badday182** [comment]
@caupolicandiaz 
- no empty line after the example header
- valid filenames other directories

**Comment by badday182** [comment]
@caupolicandiaz 
- no empty line after the example header
- valid filenames other directories

**Review by caupolicandiaz — APPROVED** [review]
@badday182 This is ready for a second review.

File: content/luau/concepts/tables/terms/getn/getn.md
Comment (KTom101) [inline content/luau/concepts/tables/terms/getn/getn.md:15]: The note is supposed to go here with a blockquote (change added below 👇 )
```suggestion
The **`.getn()`** function in Luau returns the number of elements in a table by iterating over the table and counting the elements, while ignoring any nil elements. This function is a handy tool for various table-related tasks, including checking the table's length, allocating memory for a table, and determining if a table is empty.

> **Note:** This function has been deprecated. Applying the `.getn()` function will result in an error. Use `#t` instead.
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@badday182 I added changes

**Comment by badday182** [comment]
@KTom101 Thank you for your response, I've added your comments into description

**Review by KTom101 — APPROVED** [review]

**Comment by caupolicandiaz** [comment]
@badday182 This entry is currently failing our formatting test and can't be merged until it is resolved. Often, we take responsibility for running the formatter to update entries. However, we don't have access to your remote. Contributors are asked to use the **Allow edits from maintainers** option. Please update your access settings or run the formatter on your own. Thanks

**Comment by badday182** [comment]
@caupolicandiaz I've given maintainers access to editing

**Comment by KTom101** [comment]
@badday182 Thank you. I was able to format the failing [checks](https://www.codecademy.com/resources/docs/contribution-guide/getting-set-up-on-your-computer#:~:text=There%20are%20continuous%20integration%20(CI)%20workflows) using `npm run format:all`. I will go ahead and merge this.

**Pull request merged**
